### PR TITLE
Feature/swift package manager

### PR DESCRIPTION
**Pull request by behzadardehei**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMajor(from: "5.0.0")),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(name: "RxOptional", dependencies: []),
```
Comment (tonyarnold) [inline Package.swift]: This target needs to declare "RxSwift" as a dependency.

Comment (tonyarnold) [reply Package.swift]:
```suggestion
        .target(name: "RxOptional", dependencies: ["RxSwift"]),
```

Comment (behzadardehei) [reply Package.swift]: I completely agree with you, Thank you very much.

**Comment by behzadardehei** [comment]
Would you help me, How can I do it? @tonyarnold

**Comment by tonyarnold** [comment]
@ashfurrow I would imagine so, yes.

**Comment by ashfurrow** [comment]
Cool! I've tagged this commit as `4.0.1`, let me know how it goes.

**Pull request merged**